arlo-client/public/sample_jurisdiction_filesheet.csv

```diff
@@ -0,0 +1,4 @@
+"Jurisdiction","Admin Email"
+"LaSalle Twp","admin@lasalletwp.com"
+"Detroit","lbeech@detroitelections.gov"
+"Ann Arbor","elections@annarbor.mi.us"
```

Comment:
Are these real? If so, change them to not be real?

Reply:
Good question. That's just the one @mcchilders uploaded. I'll edit it to be obviously fake.